lib/statix.ex

```diff
@@ -365,4 +367,18 @@ defmodule Statix do
       {_p1, _p2} -> [part1, ?., part2, ?.]
     end
   end
+
+  defp add_global_tags(options) do
+    global_tags = Application.get_env(:statix, :tags)
+    options_tags = Keyword.get(options, :tags)
+
+    new_tags = cond do
+      options_tags && global_tags -> options_tags ++ global_tags
+      global_tags -> global_tags
+      true -> nil
+    end
+
+    if new_tags, do: Keyword.put(options, :tags, new_tags), else: options
```

Comment:
Not sure if @lexmag would agree to this but if so I think a better implementation would be:

```elixir
global_tags = Application.get_env(:statix, :tags, [])
Keyword.update(options, :tags, [], &Keyword.merge(global_tags, &1))
```

However you need to handle global backend-specific tags, that is, what would be configured by

```elixir
config :statix, MyApp.Statix,
  tags: ...
```